unstructured/ingest/logger.py

```diff
@@ -0,0 +1,11 @@
+import logging
+
+logger = logging.getLogger("unstructured.ingest")
+handler = logging.StreamHandler()
+formatter = logging.Formatter("%(asctime)s %(processName)-10s %(levelname)-8s %(message)s")
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+
+
+def set_ingest_logging_level(level: int) -> None:
+    logger.setLevel(level)
```

Comment:
```suggestion
def ingest_log_streaming_init(level: int) -> None:
    handler = logging.StreamHandler()
    formatter = logging.Formatter("%(asctime)s %(processName)-10s %(levelname)-8s %(message)s")
    handler.setFormatter(formatter)
    logger.addHandler(handler)
    logger.setLevel(level)
```

As is written, any outside application that imports anything under `unstructured.ingest.connector` would have this stream handler and formatter added which may not be desirable (if they wanted different log handling, they'd have to remove the ones created by virtue of this `logger` module being imported).

But, I think everything is fine if the handler / formatter (i.e. everything except the `logger` object)  is just moved into this function. I also suggested a tweak on the function name to reflect the update.

Reply:
I agree that the additional flexibility is desirable, albeit niche. I've made the change as suggested.